assets/mysql-server.cnf

```diff
@@ -0,0 +1,15 @@
+[mysqld]
```

Comment:
This is a configuration file for mysql, but we're never running mysql/mariadb inside the xdmod containers. The database service in the k8s configuration in this repository is provided by the upstream mariadb image.

Reply:
The xdmod folks are running mariadb out of the same container that xdmod is running.  The example that I am working from used a separate container to run mariadb.  I went with running mariadb in a separate container as that made more sense, though it is a bit more difficult to setup.

Reply:
Actually assets/mysql-server.cnf and the vi configuration files are not being used, though they may be useful to include in the development container.